boards/arm/swan_r5/doc/index.rst

```diff
@@ -0,0 +1,241 @@
+
+Supported Features
+==================
+
+The Zephyr Swan board configuration supports the following
+hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| USB       | on-chip    | usb                                 |
++-----------+------------+-------------------------------------+
+| ADC       | on-chip    | adc                                 |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/swan_r5/swan_r5_defconfig``
+
+
+Connections and IOs
+===================
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- A0 : PA3
+- A1 : PA1
+- A2 : PC3
+- A3 : PC1
+- A4 : PC4
+- A5 : PC5
+- D4 : PE3
+- D5 : PE11
+- D6 : PE9
+- D9 : PD15
+- D10 : PA4
+- D11 : PA7
+- D12 : PA6
+- D13 : PA5
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PD6
+- UART_3_TX : PB10
+- UART_3_RX : PB11
+- LPUART_TX : PG7
+- LPUART_RX : PG8
+- I2C_1_SCL : PB6
+- I2C_1_SDA : PB7
+- I2C_2_SCL : PF1
+- I2C_2_SDA : PF0
+- I2C_3_SCL : PC0
+- I2C_3_SDA : PC9
+- SPI_1_NSS : PA4
+- SPI_1_SCK : PA5
+- SPI_1_MISO : PA6
+- SPI_1_MOSI : PA7
+- SPI_2_NSS : PD0
+- SPI_2_SCK : PD1
+- SPI_2_MISO : PB14
+- SPI_2_MOSI : PB15
+- SPI_3_NSS : PA15
+- SPI_3_SCK : PC10
+- SPI_3_MISO : PC11
+- SPI_3_MOSI : PC12
+- PWM_2_CH1 : PA0
+- USER_PB : PC13
+- LD1 : PE2
+- USB DM : PA11
+- USB DP : PA12
+- ADC1 : PA1
+
+System Clock
+------------
+
+Swan System Clock could be driven by internal or external
+oscillator, as well as main PLL clock. By default, the System clock is
+driven by the PLL clock at 80MHz, driven by a 16MHz high speed
+internal oscillator. The clock can be boosted to 120MHz if boost mode
+is selected.
```

Comment:
`The clock can be boosted to 120MHz if boost mode
is selected.`
Please remove as there is no dedicated Zephyr config to do this yet